### ISSUE TITLE
Box drawing characters have spaces in PuTTY

### ISSUE THREAD

**Issue by DannyBen** [JetBrains/JetBrainsMono#349]
## Issue

Box drawing characters have spaces when used in PuTTY on Windows.
Note that:

- Other fonts (including Fira Code) shows as expected in PuTTY
- JetBrains Mono shows as expected in other places (e.g. Sublime Text)

![image](https://user-images.githubusercontent.com/2405099/97151056-a9aa7600-1777-11eb-84d4-eadb05696f5a.png)

![image](https://user-images.githubusercontent.com/2405099/97151065-aca56680-1777-11eb-80bf-e37cc60f1a8c.png)

## Reproduction script

```bash
#!/usr/bin/env bash

# ┘ ┐ ┌ └ ┤ ┴ ┬ ├ ─ │ ┼
# ╝ ╗ ╔ ╚ ╣ ╩ ╦ ╠ ═ ║ ╬

echo " │       │   │  │      │   │"
echo "─┘ ─┐ ┌─ └─ ─┤ ─┴─ ─┬─ ├─ ─┼─"
echo "    │ │      │      │  │   │"
echo
echo " ║       ║   ║  ║      ║   ║"
echo "═╝ ═╗ ╔═ ╚═ ═╣ ═╩═ ═╦═ ╠═ ═╬═"
echo "    ║ ║      ║      ║  ║   ║"
```

## Environment

**JetBrains Mono Version**: 2.001 and 2.200
**Software**: PuTTY
**OS**: Windows 10

**Comment by philippnurullin** [comment]
Hi @DannyBen Can you please try to reproduce this problem on the latest version 2.211? https://github.com/JetBrains/JetBrainsMono/tree/master/fonts

**Comment by DannyBen** [comment]
I can confirm, this seems to be **fixed** with 2.211 (tested the Regular weight only):

![image](https://user-images.githubusercontent.com/2405099/99237479-32f61b00-2801-11eb-9110-f0c59a5e45d1.png)

Thanks for the quick fix, I keep being surprised by how much this font improves my quality-of-life :)

**Comment by philippnurullin** [comment]
Glad to hear! 
The fonts have the same vertical metrics so it must work like this in every weight.
Feel free to reopen in the problem appears again.
